.github/workflows/test.yml

```diff
@@ -121,6 +121,17 @@ jobs:
           cache-to: type=gha,mode=max
           build-args: |
             BASE_IMG_TAG=debug
+      -
+        name: Build e2e init image
+        uses: docker/build-push-action@v3
+        with:
+          load: true
+          context: .
+          file: tests/e2e/chain_init/chain-init.Dockerfile
```

Comment:
Why do we need to build this now?

Reply:
If we skip the upgrade shouldn't we need to initialize with the same version we run? If you are asking why we didn't do this before, I am unsure